File: python/iniciante/1048.py
'''
title: Aumento de Salário
number of the problem: 1048
date: 05/03/2018
author: ltsdw
'''


if __name__ == '__main__':

    s: float = float(input(''))
    sr: float
    r: float

    if s <= 400.00:
        sr = s + ((s * 15) / 100)
        r = (s * 15) / 100

        print(f'''Novo salario: {sr:.2f}\nReajuste ganho: {r:.2f}\nEm percentual: 15 %''')
    elif s <= 800.00:
        sr = s + ((s * 12) / 100)
        r = (s * 12) / 100

        print(f'''Novo salario: {sr:.2f}\nReajuste ganho: {r:.2f}\nEm percentual: 12 %''')
    elif s <= 1200.00:
        sr = s + ((s * 10) / 100)
        r = (s * 10) / 100
        print(f'''Novo salario: {sr:.2f}\nReajuste ganho: {r:.2f}\nEm percentual: 10 %''')
    elif s <= 2000.00:
        sr = s + ((s * 7) / 100)
        r = (s * 7) / 100

        print(f'''Novo salario: {sr:.2f}\nReajuste ganho: {r:.2f}\nEm percentual: 7 %''')
    else:
        sr = s + ((s * 4) / 100)
        r = (s * 4) / 100

        print(f'''Novo salario: {sr:.2f}\nReajuste ganho: {r:.2f}\nEm percentual: 4 %''')

File: haskell/iniciante/1048.hs
-- title: Aumento de Salário
-- number of the problem: 1048
-- date: 23/09/20
-- author: ltsdw

import Text.Printf

main :: IO ()
main = do
    salary' <- getLine
    let salary = read salary' :: Double
    if salary <= 400.00
        then do
            printf "Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: 15" (salary + ((salary * 15) / 100)) ((salary * 15) / 100)
            putStrLn " %"
    else if salary <= 800.00
        then do
            printf "Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: 12" (salary + ((salary * 12) / 100)) ((salary * 12) / 100)
            putStrLn " %"
    else if salary <= 1200.00
        then do
            printf "Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: 10" (salary + ((salary * 10) / 100)) ((salary * 10) / 100)
            putStrLn " %"
    else if salary <= 2000.00
        then do
            printf "Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: 7" (salary + ((salary * 7 ) / 100)) ((salary * 7 ) / 100)
            putStrLn " %"
    else do
        printf "Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: 4" (salary + ((salary * 4 ) / 100)) ((salary * 4 ) / 100)
        putStrLn " %"

